.devcontainer/devcontainer.json

```diff
@@ -59,8 +59,8 @@
 	// "forwardPorts": [5432],
 
 	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": ".devcontainer/postCreate.sh"
+	"postCreateCommand": ".devcontainer/postCreate.sh",
 
 	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
-	// "remoteUser": "root"
+	"remoteUser": "root"
```

Comment:
Why did you need this?

Reply:
When setting up the repo, the first issue I had was that apparently I didn't have enough permissions on the devcontainer to run the server. This might be due to using Dev Containers on Windows... I forgot to revert before committing!